app/controllers/list_controller.rb

```
@@ -0,0 +1,43 @@
+class ListController < ApplicationController
+  def index
+    @lists = List.all
+  end
+
+  def edit
+    @list = List.find_by_id(params[:id])
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def show
+    @list = List.find_by_id(params[:id])
```

See above re: failing silently